tests/test_simultaneous_initiate.c

```diff
@@ -1204,9 +1325,32 @@ START_TEST(test_repeated_simultaneous_initiate_lost_message_repeated_messages)
         result = session_builder_process_pre_key_bundle(alice_session_builder, bob_pre_key_bundle);
         ck_assert_int_eq(result, 0);
 
+/*Bob does schnoor*/
+    	session_state *state = 0; 
+    	session_record *bob_record = 0;
```

Comment:
When you compile this, you should get a warning that you're shadowing this variable in this for loop with a variable that has the same name around line 1299 or 1300. You should probably name these differently than the variables above